elm/src/Test/Runner/Node.elm

```diff
@@ -309,33 +309,132 @@ init { processes, globs, paths, fuzzRuns, initialSeed, report, runners } _ =
     ( model, Cmd.none )
 
 
+failInit : String -> Report -> Int -> ( Model, Cmd Msg )
+failInit message report _ =
+    let
+        model =
+            { available = Dict.empty
+            , runInfo =
+                { testCount = 0
+                , globs = []
+                , paths = []
+                , fuzzRuns = 0
+                , initialSeed = 0
+                }
+            , processes = 0
+            , nextTestToRun = 0
+            , results = []
+            , testReporter = createReporter report
+            , autoFail = Nothing
+            }
```

Comment:
I’m not super happy about having to create an empty model that is never going to be used here, but it was the best I could come up with.